samples/IdentitySample.DefaultUI/Startup.cs

```diff
@@ -32,7 +32,7 @@ public void ConfigureServices(IServiceCollection services)
                 options => options.UseSqlServer(Configuration.GetConnectionString("DefaultConnection"),
                 x => x.MigrationsAssembly("IdentitySample.DefaultUI")));
 
-            services.AddIdentity<IdentityUser, IdentityRole>()
+            services.AddIdentity<IdentityUser, IdentityRole>(o => o.Stores.MaxLengthForKeys = 128)
```

Comment:
Will this require a migration for existing users?
This is kind of a breaking change, isn't it?

Reply:
Yeah, so existing apps wouldn't be using the DefaultUI, if they switch to it, they will be opting into the new 2.1 schema...

Reply:
And if they don't want the breaking change, they can set MaxLengthForKeys to 0 to turn it off and its just like before

Reply:
Ah, yes. I see now. This should be part of our template by default, shouldn't it?

Reply:
@javiercn Yes, this will be on for new projects. Once this is in and turned on for new projects we can then scaffold the up-to-date migration and we should be set for 2.1.

People with existing apps can choose to set this on with whatever value makes sense for them, or even explicitly define different lengths in the model, and then generate a migration. But this is an explicit step that they must take so we don't break any existing apps by default.